.travis.yml

```diff
@@ -8,6 +8,7 @@ env:
 install:
     - docker build -t bailador/bailador .
     - docker run -d -e 'AUTHOR_TESTING=1' --volume=$(pwd):/srv --name bailador-test bailador/bailador /bin/sleep infinity
+    - docker exec -it bailador-test zef --verbose install .
```

Comment:
Hi :-)

you removed this line, the installation, in your commit 6e6636f6e4ee6572a7c74103ca8c176cf369a77f. That's why it was missing.

It is not part of the docker image because I am using the docker image on my local box while developing as well. Whenever there's a change in my local FS i dont need to rebuild my docker image. I just start the tests in the image with a single command. :-)

From travis point of view, I think, there is no benefit nor drawback when it's actually installed during the execution of "install". The image is not persitant.

Reply:
I put it back again... I understand why it's there instead of being a part of the creation of the image.

Reply:
Aye. Going to merge it later. I cant so it in my mobile